dev/ci/docker/bionic_coq/Dockerfile

```diff
@@ -18,9 +18,12 @@ RUN apt-get update -qq && apt-get install --no-install-recommends -y -qq \
         texlive-science tipa
 
 # More dependencies of the sphinx doc
-RUN pip3 install sphinx==1.7.8 sphinx_rtd_theme==0.2.5b2 \
+RUN pip3 install wheel sphinx==1.7.8 sphinx_rtd_theme==0.2.5b2 \
```

Comment:
What is `wheel`?

Reply:
Also all the other pip packages have explicit versions, so this one should probably follow.

Reply:
If you want to install wheel, you should probably do it with `apt`.

Reply:
I thought it was already installed, but I see it is not.